### PR TITLE
apt-add-repository: add Thai translation

### PR DESCRIPTION
**Pull request by nontw**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages.th/linux/apt-add-repository.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# apt-add-repository
+
+> ควบคุมและจัดการที่อยู่ของคลัง apt
+> ข้อมูลเพิ่มเติม: <https://manpages.debian.org/latest/software-properties-common/apt-add-repository.1.html>.
+
+- เพิ่มที่หมายของคลัง apt
+
+`apt-add-repository {{repository_spec}}`
```
Comment (navarroaxel) [inline pages.th/linux/apt-add-repository.md:8]: is `repository_spec` translatable?

Comment (nontw) [reply pages.th/linux/apt-add-repository.md]: yes, I just added :).

File: pages.th/linux/apt-add-repository.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
+# apt-add-repository
+
+> ควบคุมและจัดการที่อยู่ของคลัง apt
+> ข้อมูลเพิ่มเติม: <https://manpages.debian.org/latest/software-properties-common/apt-add-repository.1.html>.
+
+- เพิ่มที่หมายของคลัง apt
```
Comment (kbdharun) [inline pages.th/linux/apt-add-repository.md:6]:
```suggestion
- เพิ่มที่หมายของคลัง apt:
```

File: pages.th/linux/apt-add-repository.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+`apt-add-repository {{ที่อยู่จำเพาะของคลัง}}`
+
+- ลบคลัง apt
```
Comment (kbdharun) [inline pages.th/linux/apt-add-repository.md:10]:
```suggestion
- ลบคลัง apt:
```

File: pages.th/linux/apt-add-repository.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+
+`apt-add-repository --remove {{ที่อยู่จำเพาะของคลัง}}`
+
+- อัพเดตข้อมูลแคชหลังจากเพิ่มคลัง apt
```
Comment (kbdharun) [inline pages.th/linux/apt-add-repository.md:14]:
```suggestion
- อัพเดตข้อมูลแคชหลังจากเพิ่มคลัง apt:
```

File: pages.th/linux/apt-add-repository.md [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+
+`apt-add-repository --update {{ที่อยู่จำเพาะของคลัง}}`
+
+- อนุญาตให้เข้าถึงซอรส์โค้ดของโปรแกรมในคลัง
```
Comment (kbdharun) [inline pages.th/linux/apt-add-repository.md:18]:
```suggestion
- อนุญาตให้เข้าถึงซอรส์โค้ดของโปรแกรมในคลัง:
```

**Comment by nontw** [comment]
Incorporate all the suggestions.

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

**Review by navarroaxel — APPROVED** [review]

**Pull request merged**